### PR TITLE
fix(unlock-protocol-com): adding state of Unlock feedback. Fixes #10759

### LINKED ISSUE #10759: State of Unlock improvements
* Use "bar char" for active locks (vs line chart)
* show period of GDP
* round GDP numbers to nearest $

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding feedback by @clemsos in #10759
localhost_3002_state

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
Currently seems to me that the bar chart takes all the space and obstruct the lines - esp 7 Days (which is the default view) and 1 month view

Screenshot 2023-01-23 at 16 52 40

I can think of two workarounds: 

1) the scale of active locks bars can be function of the total number of locks (like percentage of total locks, but that wont work so well as lock count is cumulative).
2) by default, active locks can be deactivated

 
Screenshot 2023-01-23 at 16 58 15

File: packages/networks/src/networks/arbitrum.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const arbitrum: NetworkConfig = {
   requiredConfirmations: 12,
   erc20: null,
   baseCurrencySymbol: 'ETH',
+  baseCurrencyFormat: '0,0.000',
```
Comment (julien51) [inline packages/networks/src/networks/arbitrum.ts:36]:
```suggestion
  baseCurrencyFormat: '0,0.00',
```

File: packages/networks/src/networks/gnosis.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export const gnosis: NetworkConfig = {
   erc20: null, // no default ERC20 on xdai for now
```
Comment (julien51) [inline packages/networks/src/networks/gnosis.ts:33]:
```suggestion
  erc20: null, // no default ERC20 on XDAI for now 
```

**Comment by julien51** [comment]
Closing as stale...

**Pull request closed without merging**